<launch>
  
  <node pkg="rosserial_python" type="serial_node.py" name="AI_arduino_serial">
    <param name="port" value="/dev/ttyACM1" />
  </node>
  <node pkg="foot_pressure_sensor" type="sub_pressure.py" name="measure_pressure" />


</launch>